.circleci/workflows.yml

```diff
@@ -121,6 +121,8 @@ jobs:
         ferrocene/ci/scripts/llvm_cache.py download
         ./x.py --stage 2 build library src/tools/rustdoc
         ./x.py --stage 1 build src/tools/rustdoc
+        # Build the certified subset of libcore
+        ./x.py --stage 1 build library/core --set rust.std-features='["ferrocene_certified"]'
```

Comment:
Should this be stage 1 or 2?

Reply:
it should do the right thing if you remove the stage

Reply:
It seems it is necessary. Otherwise it will try to build stage 1 std which will fail: https://app.circleci.com/pipelines/github/ferrocene/ferrocene/12537/workflows/b6ab1eaf-a431-4914-84ee-7450e6e93b70/jobs/75360